config/initializers/mandate.rb

```diff
@@ -22,6 +53,17 @@ def active_job_queue
     end
 
     def defer(*args, wait: nil, **kwargs)
+      # We need to convert the jobs to a hash before we serialize as there's no serialization format
+      # for a job. We do this here to avoid cluttering the codebase with this logic.
```

Comment:
```suggestion
      # We need to convert the jobs to a hash before we serialize as there's no serialization
      # format for a job. We do this here to avoid cluttering the codebase with this logic.
```